test/regextopcre.test.js

```diff
@@ -0,0 +1,47 @@
+// Copyright IBM Corp. 2018. All Rights Reserved.
+// Node module: loopback-connector-couchdb2
+// This file is licensed under the Apache License 2.0.
+// License text available at https://opensource.org/licenses/Apache-2.0
+
+// Comment test cases to get CI pass,
+// will recover them when CI config done
+
+'use strict';
+
+var should = require('should');
+var db;
+
+if (!process.env.COUCHDB2_TEST_SKIP_INIT) {
+  require('./init.js');
+}
+
+describe('CouchDB2 regexToPCRE', function() {
+  before(function(done) {
+    db = global.getSchema();
+    done();
```

Comment:
I don't think there is a need to use `done` here

Reply:
@shimks I think `done()` has to be called here to invoke the callback, otherwise the test will hang.

Reply:
I think @shimks is correct. I just needed to remove done from the function params along with the done on like 21. Thank you for catching this!